cmake/GoogleCloudCppCommon.cmake

```diff
@@ -18,15 +18,9 @@
 get_filename_component(GOOGLE_CLOUD_CPP_SUBPROJECT
                        "${CMAKE_CURRENT_SOURCE_DIR}" NAME)
 
-# C++ Exceptions are enabled by default, but allow the user to turn them off.
-include(EnableCxxExceptions)
```

Comment:
The CMake novice asks if this deletion changes behavior at all

Reply:
It is already added in line 278 of the top-level CMakeList.txt file.